### PR TITLE
Incident widgets

### PR DESCRIPTION
**Pull request by jbrooksuk**
Something I'm playing around with.

image

CC @cachethq/owners

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Cool.

File: bower.json [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-        "lodash": "~2.4",
-        "messenger": "~1.4.1",
-        "Sortable": "~1.0.0",
-        "animate-sass": "~0.6.2",
-        "moment": "~2.9",
-        "livestampjs": "~1.1.2",
-        "chartjs": "~1.0.1",
-        "eonasdan-bootstrap-datetimepicker": "~3.1.3"
-    }
+  "name": "Cachet",
```
Comment (GrahamCampbell) [inline bower.json]: indentation gone wrong again

File: bower.json [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
+    "jquery": "~2.1.1",
+    "jquery-minicolors": "~2.1.10",
+    "jquery-serialize-object": "~2.4.3",
+    "lodash": "~2.4",
+    "messenger": "~1.4.1",
+    "Sortable": "~1.0.0",
+    "animate-sass": "~0.6.2",
+    "moment": "~2.9",
+    "livestampjs": "~1.1.2",
+    "chartjs": "~1.0.1",
```
Comment (GrahamCampbell) [inline bower.json]: these where sorted at some point, now they're out of order

File: resources/assets/js/app.js [hunk 3]
```diff
@@ -296,6 +296,46 @@ $(function() {
...
+                            fillColor: stfColor,
+                            spotColor: sptColor,
+                            highlightSpotColor: sptColor,
+                            spotRadius: sptRadius,
+                            valueSpots: {
+                                '0:': sptColor
+                            }
+
+                        }
+
```
Comment (GrahamCampbell) [inline resources/assets/js/app.js]: cs

**Comment by joecohens** [comment]
Nice!

**Comment by jbrooksuk** [comment]
@cachethq/owners  could you pull this down and have a look now?

**Pull request merged**